src/clusterfuzz/_internal/base/tasks.py

```diff
@@ -304,15 +344,42 @@ def get_task():
     logs.log_error('Failed to get any fuzzing tasks. This should not happen.')
     time.sleep(TASK_EXCEPTION_WAIT_INTERVAL)
 
+    return None
   return task
 
 
+def get_utasks_queue():
+  return 'utasks'
```

Comment:
Is there a way to avoid this additional queue/indirection? 

Is the main reason why we have a dedicated one just because not all tasks work with batch yet? 

If we need this control on a task by task basis, perhaps we should start annotating tasks with attributes encoding the task name (which can be filtered on in Pub/Sub) ,

Reply:
Alternatively, we can just use this queue as an intermediate step, and remove it later once everything is migrated.

Reply:
Thinking more, I suppose the other benefit of having this queue is we can batch fuzz utasks this way as well with the same mechanism. I'm fine with this approach in that case, as I can't think of any other way to reduce the number of requests/Batch Jobs we make.